src/docfx/Errors.cs

```diff
@@ -697,13 +697,13 @@ public static Error Custom404Page(Document file)
             /// Html Tag value must be in allowed list
             /// </summary>
             public static Error DisallowedHTMLTag(Document file, string tag)
-                => new Error(ErrorLevel.Info, "disallowed-html", $"HTML tag '{tag}' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.", file.FilePath);
+                => new Error(ErrorLevel.Info, "disallowed-html", $"HTML tag '{tag}' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.", file.FilePath, 0, 0, 0, 0, $"tag '{tag}'");
 
             /// <summary>
             /// Html Attribute value must be in allowed list
             /// </summary>
             public static Error DisallowedHTMLAttribute(Document file, string tag, string attribute)
-                => new Error(ErrorLevel.Info, "disallowed-html", $"HTML attribute '{attribute}' on tag '{tag}' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.", file.FilePath);
+                => new Error(ErrorLevel.Info, "disallowed-html", $"HTML attribute '{attribute}' on tag '{tag}' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.", file.FilePath, 0, 0, 0, 0, $"'{attribute}' on tag '{tag}'");
```

Comment:
consider use a parser friendly format
```suggestion
                => new Error(ErrorLevel.Info, "disallowed-html", $"HTML attribute '{attribute}' on tag '{tag}' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.", file.FilePath, name: $"{tag}_{attribute}'");
```